docs/python/modelspec/programmatic/mechanisms/nmodl.rst

```diff
@@ -925,8 +931,15 @@ Description:
 
     Also note that for any argument value outside of the interpolation range,
     the returned value of a function with a TABLE will always be the value at
-    the boundary (value at ``lowest`` if argument value is smaller than ``lowest``,
-    value at ``highest`` if argument value is larger than ``highest``).
+    the boundary. In mathematical form, assuming the table interpolates values
```

Comment:
It still states that any function with a `TABLE` statement will always clamp. Which is wrong, it only clamps if the table is being used.

Reply:
Updated.